clippy_lints/src/let_underscore.rs

```diff
@@ -122,7 +122,7 @@ declare_clippy_lint! {
     /// }
     /// // Either provide a type annotation:
     /// let _: Result<u32, ()> = foo();
-    /// // …or drop the let keyword:
+    /// // …or drop the `let` keyword to fully ignore the type no matter what:
```

Comment:
I used this phrasing because the original PR did it.
#10356
> [...] Alternatively, they can drop the let keyword to truly just ignore the value no matter what.